course_access_groups/acl_backends.py

```diff
@@ -1,19 +1,72 @@
 """
 Access Control backends to implement the Course Access Groups.
 """
-import six
 
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from organizations.api import get_course_organizations
+from organizations.models import UserOrganizationMapping
+from course_access_groups.models import (
+    CourseAccessGroup,
+    GroupCourse,
+    Membership,
+)
 
-def dummy_backend(user, resource, default_has_access, options):  # pylint: disable=unused-argument
+
+def is_organization_staff(user, course):
+    """
+    Helper to check if the user is organization staff.
+
+    TODO: Handle single-site setups in which organization is not important
+    """
+    # TODO: What if a course has two orgs? data leak I guess?
+    organizations = get_course_organizations(course.id)
+    if not organizations:
+        return False
+
+    return UserOrganizationMapping.objects.filter(
+        user=user,
+        organization_id__in=[org['id'] for org in organizations],
+        is_active=True,
```

Comment:
Should we also check the user model for `user.is_active`?

Reply:
I don't think so. But I did it anyway.